### PR TITLE
Fix broken links and added short summary of key features to the UG

### PR DESCRIPTION
**Pull request by LinkesV**
Do help to check if the key features are valid

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -47,46 +53,54 @@ VendorVault is a **desktop app for managing supplier contact information and del
...
+A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data:
+
 
 ![Ui](images/intro/Ui.png)
+<br>
 
-5. Type commands in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. In the application the box that says `Enter command here ... ` is called the command box. This is where you will input in your commands. 
+
+6. The box below the command box is called the `results display box`. This is where the result of your inputted command will be displayed.
```
Comment (ngjianyi) [inline docs/UserGuide.md:101]: Would putting the results display box in backticks make the user think that it could be part of a command?

**Review by ngjianyi — COMMENTED** [review]
LGTM

**Review by CJianzhi — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -47,46 +53,54 @@ VendorVault is a **desktop app for managing supplier contact information and del
...
 
-<br>
+A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data:
+
 
 ![Ui](images/intro/Ui.png)
+<br>
 
-5. Type commands in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. In the application, the box that says **Enter command here ...** is called the command box. This is where you will input in your commands.
```
Comment (ngjianyi) [inline docs/UserGuide.md:99]: Should command box be bolded as well for standardisation?

**Review by ngjianyi — COMMENTED** [review]
LGTM

**Pull request merged**